manifests/c/ChefSoftware/Inspec/4.50.3.1/ChefSoftware.Inspec.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ChefSoftware.Inspec
+PackageVersion: 4.50.3.1
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://packages.chef.io/files/stable/inspec/4.50.3/windows/10/inspec-4.50.3-1-x64.msi
+  InstallerSha256: d0c0c759ae956a976ef0181dd6e9b278dd8bce4aee75213065da59c7b6dc8c17
+  ProductCode: '{C5C9E10A-20B2-47E4-A15D-AFEFD65121C5}'
```

```suggestion
  ProductCode: '{B61D1488-0BA8-4129-B7C1-FE537A3E5ACB}'
```
wingetcreate or YamlCreate.ps1 should automatically do this for you.

Incorrect ProductCode means that no one will be able to upgrade the application via `winget upgrade`.